templates/terraform/encoders/compute_global_network_endpoint.go.erb

```diff
@@ -0,0 +1,21 @@
+<%# The license inside this block applies to this file.
+  # Copyright 2017 Google Inc.
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+-%>
+// Network Endpoint Group is a URL parameter only, so replace self-link/path with resource name only.
+d.Set("global_network_endpoint_group", GetResourceNameFromSelfLink(d.Get("global_network_endpoint_group").(string)))
```

Comment:
I think it might be good to have this built into the template for url_param_only fields - am running into this with other resources too. Don't have to do it in this PR, just a thought